hack/flakes/flake.nix

```diff
@@ -162,9 +162,44 @@
             '';
           };
 
+          helm-list-images = buildGo122Module rec {
```

Comment:
Do we need this plugin anymore? Wont it be downloaded as go dependency of the tool?